assets/src/web-stories-block/block/transforms.js

```diff
@@ -148,32 +148,53 @@ const transforms = {
         return idBase === 'web_stories_widget';
       },
       transform: ({ instance }) => {
+        const {
+          raw: {
+            archive_link_label: archiveLinkLabel,
+            circle_size: circleSize,
+            image_alignment: imageAlignment,
+            number_of_columns: numOfColumns,
+            number_of_columns: numOfStories,
+            view_type: viewType,
+            show_title,
+            show_author,
+            show_date,
+            show_excerpt,
+            show_archive_link,
+            sharp_corners: show_sharp_corners,
+            show_image_alignment,
+            title,
+            orderby = '',
+            order = '',
+          },
+        } = instance;
```

Comment:
Does `instance.raw` always exist? There was `instance.raw?.title` being used before, indicating that it can be `null`.

Reply:
It should also exist.

Reply:
From lines above,
```
isMatch: ({ idBase, instance }) => {
        if (!instance?.raw) {
          // Can't transform if raw instance is not shown in REST API.
          return false;
        }
        return idBase === 'web_stories_widget';
      },
```
Raw will always exist.